doc/flux_to_redux.md

```diff
@@ -0,0 +1,579 @@
+- __We replace the Flux store with both a state class and a reducer.__ In Redux, we do not define a class that is responsible for handling state updates. Rather, we create the model to represent the state and a function to describe updates. Then, use them in the instantiation of a Redux `Store` object.
+- __Actions are not held in an overarching wrapper class.__ Typically in Redux, actions "stand alone" from each other. Our reducer should have cases for all the actions that exist, but unlike Flux there is no need to knit our store and actions together by actually providing an actions instance to the store. Consequently, actions are broken away, pulled directly into the component, and triggered by passing an action instance into `dispatch` - the centralized dispatcher accessible via `props` or `mapDispatchToProps`.
+- __In Redux, the entire store is not passed to components via props.__ React Redux's `connect` function uses context to allow you to access the store's state without passing it through multiple layers of components, and also provides a mechanism to map the relevant parts of state directly to props (an optionally only rerender when those props change). `connect` is paired with wrapping the component tree in a `ReduxProvider` component.
+
+### Basic Conversion Step by Step
+> NOTE: This document does not attempt to teach _how_ to use Redux. If any of these steps cause confusion on the implementation details of Redux, see the [OverReact Redux Documentation](./over_react_redux_documentation.md).
+1. __Refactor the actions.__ Remove the action container class, and replace action instances with classes.
+    ```dart
+    // Before
+    class ExampleActions {
+      final Action<void> randomizeBackgroundColor = Action();
+      final Action<String> setBackgroundColor = Action();
+    }
+     
+    // After
+    class RandomizeBackgroundColorAction {}
+    class SetBackgroundColorAction {
+      final String backgroundColor;
+      SetBackgroundColorAction(this.backgroundColor);
+    }
+    ```
+
+    If your `Action` receives a custom class as typing for the action payload, that class could be a great starting point for the creation of your new Redux action.
+
+    ```dart
+    // Before
+    class ExampleActions {
+        final Action<SetBackgroundColorPayload> randomizeBackgroundColor = Action();
+    }
+
+    class SetBackgroundColorPayload {
+        String backgroundColor;
+        SetBackgroundColorAction(this.backgroundColor);
+    }
+    
+    // After
+    class SetBackgroundColorAction {
+        final String backgroundColor;
+        SetBackgroundColorAction(this.backgroundColor);
+    }
+    ```
+
+1. __Pull state mutation logic out of the store and into a reducer.__ Typically within a Flux store you have a `triggerOnActionV2` call that identifies an action and a function used to respond to that action. In many cases, that same logic should be perfect for a reducer.
+    ```dart
+    // Before transition
```

Comment:
I think we should show the necessary imports for clarity as to what the `flux` namespace represents.

```suggestion
    import 'package:w_flux/w_flux.dart' as flux;

    // Before transition
```